examples/src/bin/psram.rs

```diff
@@ -23,22 +33,18 @@ fn init_psram_heap() {
 #[entry]
 fn main() -> ! {
     #[cfg(debug_assertions)]
-    compile_error!("PSRAM on ESP32-S3 needs to be built in release mode");
-
-    #[cfg(feature = "log")]
-    esp_println::logger::init_logger_from_env();
+    compile_error!("PSRAM example must be built in release mode!");
 
     let peripherals = Peripherals::take();
+
     psram::init_psram(peripherals.PSRAM);
     init_psram_heap();
 
     let system = peripherals.SYSTEM.split();
-    let _clocks =
-        ClockControl::configure(system.clock_control, esp_hal::clock::CpuClock::Clock240MHz)
-            .freeze();
+    let _clocks = ClockControl::configure(system.clock_control, CpuClock::Clock240MHz).freeze();
```

Comment:
Wouldn't it be better to use `ClockControl::max`?

Reply:
Why is that better at demonstrating how to use PSRAM?

Reply:
I meant more flexible, not “better”. Instead of having a hard-coded value that's not relevant for the example and might not be compatible for all the targets in the future.

Still, this is quite a nitpick comment, so feel free to disregard it.